test/built-ins/Proxy/getOwnPropertyDescriptor/result-is-undefined-targetdesc-is-undefined.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.5
+description: >
+    Throws a TypeError exception if trap result is undefined and target property
+    descriptor is undefined.
+info: >
+    [[GetOwnProperty]] (P)
+
+    ...
+    14. If trapResultObj is undefined, then
+        a. If targetDesc is undefined, return undefined.
+    ...
+---*/
+
+var t = {};
+var p = new Proxy(t, {
+    getOwnPropertyDescriptor: function(target, prop) {
+        return;
+    }
+});
+
+assert.sameValue(
+    typeof Object.getOwnPropertyDescriptor(p, "foo"),
```

Comment:
No need to use the `typeof` operator, you can assert equality with `undefined` directly. The second assertion is redundant. It's also a good idea to make sure the handler was invoked (since otherwise this test will pass if the trap is completely unimplemented)